pootle/apps/pootle_store/unit/results.py

```diff
@@ -22,6 +22,11 @@
 class UnitResult(UnitProxy):
 
     @property
+    def nplurals(self):
+        return self.unit[
+            "store__translation_project__language__nplurals"] or 0
```

Comment:
i think the default is correct - im just trying prevent ``None`` from being passed